source/core/ut_expectation_processor.pkb

```diff
@@ -88,7 +88,12 @@ create or replace package body ut_expectation_processor as
   begin
     add_expectation_result(ut_expectation_result(ut_utils.gc_failure, null, a_message));
   end;
-
+  
+  procedure report_failure_no_caller(a_message in varchar2) is
```

Comment:
Why do we need this?

Reply:
Good catch, to fast push, didn't cleanup.